src/lib/components/TagTypeSelection.svelte

```
@@ -0,0 +1,45 @@
+				{#each selectedTagtypes as tagtype}
+					{@render myBadge(tagtype)}
+				{/each}
+			</span>
+		{/if}
+	</Select.Trigger>
+	<Select.Content>
+		<Select.Group>
+			<Select.GroupHeading>Select Tagtypes</Select.GroupHeading>
+			{#each tagtypeList as tagtype}
+				<Select.Item value={String(tagtype.id)} label={tagtype.name} />
+			{/each}
```

ESLint

[error] Each block should have a key

(svelte/require-each-key)